From 155ddd9f5f51091730c26232afe81d6de9944611 Mon Sep 17 00:00:00 2001
From: Misael Lopez Cruz <misael.lopez@ti.com>
Date: Fri, 17 Dec 2010 19:47:13 -0600
Subject: ASoC: sdp4430: Enable DL backends for MM_DL_LP port

Multimedia DL low-power port can also be used with other backends
(PDM_DL1, PDM_DL2, MM_EXT0, BT_VX). Low-power port is now only
restricted to ping-pong buffer, not to OPP25 anymore.

Change-Id: Icbfa22afa2322563963ff98a14db5cc0f5be2296
Signed-off-by: Misael Lopez Cruz <misael.lopez@ti.com>
Signed-off-by: Chris Kelly <c-kelly@ti.com>
---
 sound/soc/omap/sdp4430.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

Index: Kernel/sound/soc/omap/sdp4430.c
===================================================================
--- Kernel.orig/sound/soc/omap/sdp4430.c	2011-01-19 00:10:05.102071740 +0530
+++ Kernel/sound/soc/omap/sdp4430.c	2011-01-19 00:14:57.113321326 +0530
@@ -465,6 +465,9 @@
 
 static const char *mm_lp_be[] = {
 		OMAP_ABE_BE_PDM_DL1,
+		OMAP_ABE_BE_PDM_DL2,
+		OMAP_ABE_BE_BT_VX,
+		OMAP_ABE_BE_MM_EXT0,
 };
 
 /* Digital audio interface glue - connects codec <--> CPU */
